drivers/hmis/app/graphql/types/hmis_schema/referral_posting.rb

```diff
@@ -22,6 +22,7 @@ class HmisSchema::ReferralPosting < Types::BaseObject
     # Fields that come from ReferralHouseholdMembers
     field :hoh_name, String, null: false
     field :hoh_mci_id, ID, null: true
+    field :hoh_client, HmisSchema::Client, null: true
```

Comment:
This was added to the schema but its not used by https://github.com/greenriver/hmis-frontend/pull/480, is that right?

Reply:
Yes, it looks like we go through hoh_enrollment->client instead so this isn't needed. We can remove it

Reply:
I'll leave it for now, I think we will probably want it later. thanks